### DISCUSSION TITLE
组件返回了markdown内容，锚点无法识别到，且无法自动渲染，被识别为纯字符串

### DISCUSSION THREAD

**Discussion by rookie-max** [umijs/dumi#2142] [Q&A]
我在使用过程中，markdown里面的某些内容来自于第三方markdown，我使用组件获取，并将内容返回，内容不能自动渲染成markdown的内容，也无法生成对应的锚点，当然全局搜索肯定是无法支持

**Comment by PeachScript** [comment]
dumi 不支持运行时的 Markdown 动态加载及编译

三方 Markdown 如果是静态内容，可以构建前用脚本拉到本地，可以参考 dumi 如何拉 Umi 的文档配置：https://github.com/umijs/dumi/blob/85f5e116c4d878fbfb65155b3a6622ae9248f7ae/package.json#L47
